README.md

```diff
@@ -7,31 +7,49 @@ about given URLs on [GOV.UK](https://www.gov.uk/).
 
 ## Requirements
 
-To run the code you will need to have at least Go 1.2 installed.
+Go 1.6.2
 
 ## Development
 
-You can run the tests locally by running the following to use
-`third_party.go` to fetch the dependencies:
+From within `$GOPATH` run `make` to run the tests and build a binary.
+Tests, build etc can also be run using the standard `go build`, `go test`...
 
-```bash
-make
-```
+## Dependencies
 
-Alternatively you can install the dependencies directly to your
-`$GOPATH` and run the tests using:
+Dependencies are vendored into `vendor` and have been committed so
+shouldn't need to be installed.
 
-```bash
-go get -v ./...
-go test -v ./...
-```
+Use [godep](https://github.com/tools/godep) to add update
+dependencies and commit to source control as they are no longer
+installed during CI.
+
+## GOV.UK Development VM
+
+The Dev VM now has a `$GOPATH` configured at `/var/govuk/gopath` so this
+app should be checked out into
+`/var/govuk/gopath/src/github.com/alphagov/metadata-api` for the `go`
+commands and `make` etc to function correctly.
+
+Avoid trying to symlink the directory into the `$GOPATH` either within
+the VM or on the host as this causes issues with dependencies in `vendor`. 
+Working with the same directory structure on the host and allowing NFS to mount 
+that into the VM works well.
 
-## Running
+For convenience using somthing like
+[direnv](https://github.com/direnv/direnv) to set up a local `$GOPATH`
+when you `cd` into the root of the repository keeps parity between the
+host and VM.
 
-You can build a binary using either `make build` or `go build`. You
-should then be able to run the binary directly.
+e.g
+
+```
+export GOPATH=<path to parent gopath dir>
+export PATH=$GOPATH/bin:$PATH
+
+```
+within `.envrc` in the `metadata-api` root.
```

Comment:
I don't usually commit it as I've always seen it as a local thing. The only reasons I can think of are that `.envrc` is specific to [direnv](https://github.com/direnv/direnv), as far as I know, so it would require the 'user' to be using that. Also, they may have other environment stuff that they set that is personal to their setup which would conflict with a committed `.envrc`.